neuralforecast/models/timemixer.py

```diff
@@ -200,13 +201,13 @@ def __init__(self, seq_len, down_sampling_window, down_sampling_layers):
             [
                 nn.Sequential(
                     torch.nn.Linear(
-                        seq_len // (down_sampling_window ** (i + 1)),
-                        seq_len // (down_sampling_window**i),
+                        math.ceil(seq_len / (down_sampling_window ** (i + 1))),
```

Comment:
I tried using math.ceil on my end as well but it seems unable to solve all the edge cases, due to the variant output shape of previous conv / pooling modules. Currently I'm trying to work on a more stable solution by pre-calculating the output shape of those layers and pass them through down the line...

Reply:
I guess we should stick to using the average as the down sampling method, since configuration of the model for the paper only uses that anyway. That will avoid the edge cases of shapes mismatch.